modules/interactivity.js

```diff
@@ -0,0 +1,29 @@
+const items = JSON.parse(localStorage.getItem('TodoList')) || [];
+
+const itemCheck = () => {
+  const checkboxes = document.querySelectorAll('.checkbox');
+
+  checkboxes.forEach((checkbox, index) => {
+    checkbox.addEventListener('change', (e) => {
+      if (items[index]) {
+        items[index].completed = e.target.checked;
+        localStorage.setItem('TodoList', JSON.stringify(items));
+      }
```

Comment:
- [ ] Kindly make sure when the task is checked we only change the `completed` status to `true` or `false` but now when the user clicks the task `checkbox` in your app it changes the task description to static word `on`. Kindly fix that.

    - ### YOUR APP BEHAVIOUR
     ![chrome-capture-2022-10-12 (9)](https://user-images.githubusercontent.com/87197412/201489291-c13d0e80-52c6-4336-93ee-4d8ada29e557.gif)